### PR TITLE
console.out to know which island version is used

### PR DESCRIPTION
**Pull request by sshtel**
simply console out island versions.

any better idea?

### CODE REVIEW

**Comment by kuyh** [comment]
With this script, we can see package.json version
if package.json refers next or ^3.6.4 
we can't identify real version of installed package

**Comment by sshtel** [comment]
agree, I think we can just command `npm list | grep island`

**Comment by sshtel** [comment]
fixed to use spawnSync to command npm list. And used its JSON output!

**Comment by kuyh** [comment]
smart solution!!

File: tasks/build.js [hunk 1]
```diff
@@ -9,8 +9,17 @@ module.exports = function (gulp) {
...
+  const spawnSync = require('child_process').spawnSync;
+  const npmlist = JSON.parse(spawnSync('npm', ['list', '--depth=0', '--json']).stdout.toString());
+
   // for incremental build test
   function compileWithGulpTypescript() {
+    console.log('============= Compile Versions of Island Packs ===========');
+    console.log(`island: ${npmlist.dependencies.island.version}`);
+    console.log(`island-base: ${npmlist.dependencies['island-base'].version}`);
+    console.log(`island-keeper: ${npmlist.dependencies['island-keeper'].version}`);
+    console.log(`distlock: ${npmlist.dependencies.distlock.version || undefined}`);
```
Comment (heycalmdown) [inline tasks/build.js]: Why do we need `|| undefined`?

Comment (sshtel) [reply tasks/build.js]: uh.. my mistake. it's not necessary. Will remove it.

**Pull request merged**